.releaserc

```diff
@@ -0,0 +1,25 @@
+{
+    "branches": [
+        { "name": "main" },
+        "+([0-9])?(.{+([0-9]),x}).x",
+        { "name": "beta", "channel": "beta", "prerelease": true },
+        { "name": "alpha", "channel": "alpha", "prerelease": true },
+        { "name": "next", "channel": "next" }
```

Comment:
Is `next` not prerelease?

Reply:
https://semantic-release.gitbook.io/semantic-release/recipes/release-workflow/distribution-channels

I followed this kinda flow through it. seems like next is not considered prerelease.

Reply:
Huh, interesting. Fair enough.